recommenders/evaluation/python_evaluation.py

```diff
@@ -736,6 +736,19 @@ def get_top_k_items(
 }
 
 
+# get col_rating column
+def _get_col_rating_column(relevancy_method:str, **kwargs) -> str:
+    r"""Helper utility to simplify the arguments of eval metrics
+    Fixes -: https://github.com/microsoft/recommenders/issues/1737
+    """
+    if relevancy_method != "top_k":
+        assert "col_rating" in kwargs, "Expected an argument `col_rating` but wasn't found."
```

Comment:
A format suggestion, in the code we would normally use `raise ValueError` instead of `assert`. We prefer to use `assert` for the tests, and exceptions for the normal code.

Reply:
Thanks for the feedback, I have updated the same. LMK if it looks good now! Should have done with :(!